frameworks/keyed/ember/app/components/my-table.gjs

```diff
@@ -116,8 +116,8 @@ export default class MyTable extends Component {
           {{#each this.data key="id" as |item|}}
             <tr class={{if (eq item.id this.selected) 'danger'}}>
               <td class="col-md-1">{{item.id}}</td>
-              <td class="col-md-4"><a {{on 'click' (fn this.select item)}}>{{item.label}}</a></td>
-              <td class="col-md-1"><a {{on 'click' (fn this.remove item)}}><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></a></td>
+              <td class="col-md-4"><a onclick={{fn this.select item}}>{{item.label}}</a></td>
```

Comment:
@NullVoxPopuli didn't know you could do this..

Reply:
ye, it's very nice -- I made a little decision tree to help with deciding between native event handlers and "event handlers with cleanup (on)": https://github.com/emberjs/rfcs/pull/1007#issuecomment-1950275078

Reply:
When is cleanup necessary? Like components that get removed from view? Or ssr?

Reply:
cleanup is needed when:
- dynamic modifier is used (via a condition)
- or when the handler could change (the function itself)